comfy/cli_args.py

```diff
@@ -118,6 +118,7 @@ class LatentPreviewMethod(enum.Enum):
 parser.add_argument("--windows-standalone-build", action="store_true", help="Windows standalone build: Enable convenient things that most people using the standalone windows build will probably enjoy (like auto opening the page on startup).")
 
 parser.add_argument("--disable-metadata", action="store_true", help="Disable saving prompt metadata in files.")
+parser.add_argument("--no-custom-node", action="store_true", help="Disable loading custom nodes.")
```

Comment:
I think this should be: `--disable-custom-nodes`

Reply:
Renamed `--disable-all-custom-nodes`, as it `--disable-custom-nodes` feels like you can pass a list of nodes to disable as argument.